src/studio/src/designer/backend/Languages/ini/nb.ini

```diff
@@ -367,16 +367,17 @@ add_restriction = Legg til restriksjon
 add_root_item = Legg til rot element
 array = Liste
 boolean = Ja/Nei svar
-delete = Slett
-promote = Flytt opp
-import = Importer
 definitions = Definisjoner
+delete = Slett
+delete_field = Slett felt
 description = Beskrivelse
 descriptive_fields = Beskrivende felter
 enum = Gyldige verdier
 fields = Felter
 go_to_main_component = Gå til hovedkomponent
```

Comment:
This was deleted in the English version...

Reply:
Good catch. Removed both places.